src/Tizen/Avalonia.Tizen/NuiViewControlHandle.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Avalonia.Controls.Platform;
+using Tizen.NUI.BaseComponents;
+
+namespace Avalonia.Tizen;
+
+/// <summary>
+/// Tizen Nui native view handle for native view attachment
+/// </summary>
+public class NuiViewControlHandle : INativeControlHostDestroyableControlHandle
+{
+    internal const string ViewDescriptor = nameof(View);
```

Comment:
I think the descriptor should be a bit more specific about platform. As "View" doesn't tell anything about tizen or Nui.